src/Symfony/Component/DependencyInjection/Compiler/ValidateEnvPlaceholdersPass.php

```diff
@@ -48,7 +48,7 @@ public function process(ContainerBuilder $container)
             foreach ($resolvingBag->getEnvPlaceholders() + $resolvingBag->getUnusedEnvPlaceholders() as $env => $placeholders) {
                 $values = array();
                 if (false === $i = strpos($env, ':')) {
-                    $default = $defaultBag->has("env($env)") ? $defaultBag->get("env($env)") : null;
+                    $default = $defaultBag->has("env($env)") ? $defaultBag->get("env($env)") : '';
                     $defaultType = null !== $default ? self::getType($default) : 'string';
```

Comment:
do we still need this null check here now ?

Reply:
yep

```
parameters
    env(SOME): ~
```

that's the only way to enable nullability for envs.